packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1538,6 +1540,27 @@ export class ContainerRuntime
 			headerData.allowTombstone = request.headers[AllowTombstoneRequestHeaderKey];
 		}
 
+        const dataStoreNodePath = `/${id}`
```

Comment:
should this be under a feature flag so it can be disabled

Reply:
That's an interesting thought, so if a datastore is deleted, it's already gone from the runtime. There are feature flags that prevent datastores from getting deleted, but once they are deleted, then this call would just hang instead.